tools/biosplitmerge_example.txt

```diff
@@ -0,0 +1,39 @@
+Demonstrations of biosplitmerge, the Linux eBPF/bcc version.
+
+This tool traces bio struct's split and merge in the block layer. When a bio struct gets split, 
+it is split into two bios; while it gets merged, it is merged into an existing request struct. 
+The tool monitors the split/merge events, and emit the full data of the events when they happen.
+
+
+Usage:
+# ./biosplitmerge.py       Trace both split and merge events.
+# ./biosplitmerge.py -S    Trace split events only.
+# ./biosplitmerge.py -M    Trace merge events only.
+
+Sample output:
+TIME(s)     EVENT         COMMAND        PID    DISK    T  IN-SECTOR   IN-BYTES OUT-SECTOR  OUT-BYTES
+0.0000      Split         dd             1262   sdb     W     282624    2097152     282624    1310720
+0.0118      Split         dd             1262   sdb     W     286720    2097152     286720    1310720
+4.2098      Split         dd             1263   sdb     W     290816    2097152     290816    1310720
```

Comment:
So this read got split into two, in-bytes staying the same while in-sector doesn't is slightly confusing?

Reply:
oh, so what we're seeing is everything except the last part of the bio, since we're spitting out the split off parts, but not the remainder?

Reply:
This output format is indeed confusing. I'll improve the format to include the remainder of split bios.